### PR TITLE
chore: docs, update cloud deployment doc

### PR DESCRIPTION
**Pull request by YuriBuerov**
#### Checklist

- [ ] run `make test`
- [ ] PR title must follow [conventional-commit-standard](https://github.com/wundergraph/wundergraph/blob/main/CONTRIBUTING.md#conventional-commit-standard)
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message and code follows the [Code of conduct](https://github.com/wundergraph/wundergraph/blob/main/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: docs-website/src/pages/docs/cloud/deployments.md [hunk 1]
```diff
@@ -10,29 +10,49 @@ WunderGraph Cloud is currently in private alpha. If you haven't done so already,
...
+# Deploying monorepo projects to the WunderGraph Cloud
+
+WunderGraph now supports the deployment of monorepo projects. To be recognized as a monorepo by the WunderGraph Cloud,
+your repository must meet the following criteria:
+
+1. The root of the repository must contain a `package.json` file and the corresponding lock file (`pnpm-lock.yaml`, `yarn.lock`, or `package-lock.json`).
+2. Workspaces are configured in the root `package.json` file or `pnpm-workspace.yaml` file.
+3. The WunderGraph project itself should not contain its own lock file, otherwise it will be deployed as a standalone project.
+
+If the repository meets these requirements, the WunderGraph Cloud will check for a build command in the root `package.json` file
```
Comment (StarpTech) [inline docs-website/src/pages/docs/cloud/deployments.md:51]: Will it always look for the build script?

File: docs-website/src/pages/docs/cloud/deployments.md [hunk 2]
```diff
@@ -10,29 +10,49 @@ WunderGraph Cloud is currently in private alpha. If you haven't done so already,
...
+and execute it during the deployment process.
+
+The WunderGraph Cloud officially supports the `turborepo`, which is a type of monorepo that allows for faster and more efficient builds.
+If your repository is set as a `turborepo`, no additional steps are required for deployment.
```
Comment (StarpTech) [inline docs-website/src/pages/docs/cloud/deployments.md:55]: How do I set a repository as turborepo?

File: docs-website/src/pages/docs/cloud/deployments.md [hunk 3]
```diff
@@ -10,29 +10,49 @@ WunderGraph Cloud is currently in private alpha. If you haven't done so already,
...
+The WunderGraph Cloud will automatically recognize and deploy your `turborepo` project.
+
+To test this feature, you can fork the [turborepo example repository](https://github.com/wundergraph/turbo-wunder-svelte/fork).
```
Comment (StarpTech) [inline docs-website/src/pages/docs/cloud/deployments.md:58]: Could we create a dedicated one e.g `cloud-starter-turbo` ?

**Review by StarpTech — CHANGES_REQUESTED** [review]

File: docs-website/src/pages/docs/cloud/deployments.md [hunk 4]
```diff
@@ -10,29 +10,49 @@ WunderGraph Cloud is currently in private alpha. If you haven't done so already,
...
+your repository must meet the following criteria:
+
+1. The root of the repository must contain a `package.json` file and the corresponding lock file (`pnpm-lock.yaml`, `yarn.lock`, or `package-lock.json`).
+2. Workspaces are configured in the root `package.json` file or `pnpm-workspace.yaml` file.
+3. The WunderGraph project itself should not contain its own lock file, otherwise it will be deployed as a standalone project.
+
+If the repository meets these requirements, the WunderGraph Cloud will check for a build command in the root `package.json` file
+and execute it during the deployment process.
+
+The WunderGraph Cloud officially supports the `turborepo`, which is a type of monorepo that allows for faster and more efficient builds.
```
Comment (jensneuse) [inline docs-website/src/pages/docs/cloud/deployments.md:54]: supports {remove "the"} `turborepo`

File: docs-website/src/pages/docs/cloud/deployments.md (hunk 1 above)
Comment (jensneuse) [inline docs-website/src/pages/docs/cloud/deployments.md:51]: requirements, {remove "the"} WunderGraph Cloud

**Review by jensneuse — CHANGES_REQUESTED** [review]

File: docs-website/src/pages/docs/cloud/deployments.md (hunk 2 above)
Comment (YuriBuerov) [reply docs-website/src/pages/docs/cloud/deployments.md:55]: Is it a part of our documentation?

File: docs-website/src/pages/docs/cloud/deployments.md (hunk 1 above)
Comment (YuriBuerov) [reply docs-website/src/pages/docs/cloud/deployments.md:51]: we are in monorepo deployment section, so in this case - yes

File: docs-website/src/pages/docs/cloud/deployments.md (hunk 2 above)
Comment (StarpTech) [reply docs-website/src/pages/docs/cloud/deployments.md:55]: It was more of a question. I don't understand this section. How can a Use set a repository as `turborepo`?

Comment (YuriBuerov) [reply docs-website/src/pages/docs/cloud/deployments.md:55]: what would you suggest?

Comment (StarpTech) [reply docs-website/src/pages/docs/cloud/deployments.md:55]: Maybe 

> If you have a `turbo.json` at the root of your repository, no additional steps are required for the deployment. We will build the repository with `turbo run build`.

**Comment by StarpTech** [comment]
@YuriBuerov What about this PR?

**Pull request closed without merging**